app/src/main/python/gogdl/xdelta/objects.py

```
@@ -0,0 +1,139 @@
+class HalfInstruction:
+    type: int = 0
+    size: int = 0
+    addr: int = 0
+
+
+@dataclass
+class AddressCache:
+    s_near = CodeTable.near_modes
+    s_same = CodeTable.same_modes
+    next_slot = 0
+    near_array = [0 for _ in range(s_near)]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)